app/src/atoms/SelectField/index.tsx

```diff
@@ -0,0 +1,99 @@
+import * as React from 'react'
+import find from 'lodash/find'
+import { Select } from './Select'
+import { COLORS, Flex, TYPOGRAPHY } from '@opentrons/components'
+import { css } from 'styled-components'
+
+import type { SelectProps, SelectOption } from './Select'
+import type { ActionMeta, MultiValue, SingleValue } from 'react-select'
+
+export interface SelectFieldProps {
+  /** optional HTML id for container */
+  id?: SelectProps['id']
+  /** field name */
+  name: NonNullable<SelectProps['name']>
+  /** react-Select option, usually label, value */
+  options: NonNullable<SelectProps['options']>
+  /** currently selected value */
+  value: string | null
+  /** disable the select */
+  disabled?: SelectProps['isDisabled']
+  /** optional placeholder  */
+  placeholder?: SelectProps['placeholder']
+  /** menuPosition prop to send to react-select */
+  menuPosition?: SelectProps['menuPosition']
+  /** render function for the option label passed to react-select */
+  formatOptionLabel?: SelectProps['formatOptionLabel']
+  /** optional caption. hidden when `error` is given */
+  caption?: React.ReactNode
+  /** if included, use error style and display error instead of caption */
+  error?: string | null
+  /** change handler called with (name, value, actionMeta) */
+  onValueChange?: (
+    name: string,
+    value: string,
+    actionMeta: ActionMeta<SelectOption>
+  ) => void
+  /** blur handler called with (name) */
+  onLoseFocus?: (name: string) => void
+  /** optional prop to make select not searchable through typing into the search field  */
+  isSearchable?: boolean
+  /** optional width to specify the width of the select field and dropdown menu */
+  width?: string
+}
+
+const CAPTION_STYLE = css`
+  font-size: ${TYPOGRAPHY.fontSizeCaption};
+  &.error {
+    color: ${COLORS.error};
+    font-weight: ${TYPOGRAPHY.fontWeightSemiBold};
+  }
+`
+
+export function SelectField(props: SelectFieldProps): JSX.Element {
+  const {
+    id,
+    name,
+    options,
+    disabled,
+    placeholder,
+    menuPosition,
+    formatOptionLabel,
+    error,
+    onValueChange,
+    onLoseFocus,
+    isSearchable,
```

Comment:
Maybe set the default value
```suggestion
    isSearchable = false,
```

Reply:
Don't we want the network selection `select` to have `isSearchable` = `true`? I am not sure if I should default this to false or not.